src/structures/GuildMember.js

```diff
@@ -284,6 +302,18 @@ class GuildMember extends Base {
     return this.edit({ nick }, reason);
   }
 
+  
+  /**
+   * Disables or enables communication for this member.
+   * @param {?Date} The duration in milliseconds the guild member should have communication disabled for, or `null` to enable communication for them
```

Comment:
This probably crosses the length limit, you should consider splitting into multiple lines
```suggestion
   * @param {number|null} seconds The duration in seconds the guild member should have communication disabled for, or `null` to enable communication for them
```